client/main.lua

```diff
@@ -1,51 +1,54 @@
 local config = require 'config.client'
 
 local function convert(tbl)
-    if tbl.items then
-        local items = {}
-        for _, v in pairs(tbl.items) do
-            items[#items + 1] = convert(v)
+    local _, e = pcall(tbl.enableMenu)
+    if type(e) == "string" or e then
```

Comment:
Why this?

Reply:
enableMenu must return bool

Reply:
This solves the problem that radialmenu will not work if this code is not on this line and enableMenu is not configured